Makefile

```diff
@@ -166,7 +166,11 @@ _client-test-mocha:  ## Run mocha tests via karma
 _client-test-qunit:  ## Run qunit tests via karma
```

Comment:
Remove the now unused `_client-test-mocha` and `_client-test-qunit` Makefile targets?

Reply:
Sure.